src/cli/getFacetsFromIndex.js

```diff
@@ -0,0 +1,17 @@
+const algoliasearch = require('algoliasearch');
+
+module.exports = async function getFacetsFromIndex({
+  appId,
+  apiKey,
+  indexName,
+} = {}) {
+  const client = algoliasearch(appId, apiKey);
+  const index = client.initIndex(indexName);
```

Comment:
We use these in [`getAttributesFromIndex`](https://github.com/algolia/create-instantsearch-app/blob/master/src/cli/getAttributesFromIndex.js) as well, maybe we could instantiate those earlier and pass the client or the index to both functions directly.

Even better would be to get all the needed data from Algolia in a multi-query, so we reduce the network calls.